octobot/__init__.py

```diff
@@ -14,6 +14,50 @@
 #  You should have received a copy of the GNU Lesser General Public
 #  License along with this library.
 
+from __future__ import print_function
+from distutils.version import LooseVersion
+import os
+import sys
+
+from octobot_commons.enums import PlatformsName
+from octobot_commons.os_util import get_os
+
+MIN_PYTHON_VERSION = (3, 7)
+MIN_TENTACLE_MANAGER_VERSION = "1.0.10"
+
+# check python version
+current_version = sys.version_info
+if not current_version >= MIN_PYTHON_VERSION:
+    print("OctoBot requires a Python version to be higher or equal to Python " + str(MIN_PYTHON_VERSION[0])
+          + "." + str(MIN_PYTHON_VERSION[1]) + " current Python version is " + str(current_version[0])
+          + "." + str(current_version[1]) + "\n"
+          + "You can download Python last versions on: https://www.python.org/downloads/", file=sys.stderr)
+    sys.exit(-1)
```

Comment:
this code has to be the 1st executed lines, before any import (otherwise it's useless, python2.7 will say syntax error at the 1st fstring)

Reply:
Oh yeah you're right

Reply:
But it's already the case, the checking code needs these imports.